students/yixingxu/lesson03/Slicing_Exercise.py

```diff
@@ -0,0 +1,57 @@
+def exchange_first_last(seq):
+	# the first and last items exchanged
+	first = seq[:1]
+	mid = seq[1:-1]
+	last = seq[-1:]
+	a_new_sequence = last + mid + first
+	return a_new_sequence
+
+def remove_every_other_item(seq):
+	# every other item removed
+	a_new_sequence = seq[::2]
+	return a_new_sequence
+	
+def four_letters_exchange_remove(seq):
+	#with the first 4 and the last 4 items removed, and then every other item in between
+	a_new_sequence = seq[4:-4:2]
+	return a_new_sequence	
+	
+def reverse_seq(seq):
+	#with the elements reversed (just with slicing).
+	a_new_sequence = seq[-1::-1]
+	return a_new_sequence	
+	
+def mid_last_first(seq):
+	# with the middle third, then last third, then the first third in the new order.
+	first_third = seq[:len(seq)//3]
+	mid_third = seq[len(seq)//3:-len(seq)//3]
+	last_third = seq[-len(seq)//3:]
+	a_new_sequence = mid_third + last_third + first_third
+	return a_new_sequence
```

Comment:
make sure indent is set to 4 spaces (no tabs)

I noticed later assignments had good indentation so it might be something you have fixed already 🙂